gix-blame/src/types.rs

```diff
@@ -334,6 +345,17 @@ pub struct UnblamedHunk {
 }
 
 impl UnblamedHunk {
+    /// Create a new instance
+    pub fn new(range: Range<u32>, suspect: ObjectId) -> Self {
```

Comment:
Since, in the past, we often got confused by the distinction between `blamed_file` and `source_file`, what about naming this `from_range_in_blamed_file`?